azure-pipelines.yml

```diff
@@ -253,145 +181,100 @@ jobs:
+      arguments: '-BuildRoot $(buildOutDir)\LocalhostWebServer -StaticFileRoot $(Agent.TempDirectory)\TestLocalIndex -CertPath $(HTTPSDevCert.secureFilePath) -CertPassword microsoft'
 
   - task: CopyFiles@2
-    displayName: 'Copy x64 Files to Package Output'
+    displayName: 'Copy Files to Package Output'
     inputs:
-      SourceFolder: '$(system.defaultWorkingDirectory)\src\x64\Release\WindowsPackageManager'
-      TargetFolder:  '$(system.defaultWorkingDirectory)\src\AppInstallerCLIPackage\bin\x64\Release'
+      SourceFolder: '$(buildOutDir)\WindowsPackageManager'
+      TargetFolder:  'src\AppInstallerCLIPackage\bin\$(buildPlatform)\$(buildConfiguration)'
       Contents: WindowsPackageManager.dll
       CleanTargetFolder: false
       OverWrite: true
-    condition: succeededOrFailed()
 
   - task: VSTest@2
-    displayName: Run E2E Tests Packaged x64
+    displayName: Run E2E Tests Packaged
     inputs:
-      testRunTitle: 'E2E Packaged x64'
+      testRunTitle: 'E2E Packaged'
       testSelector: 'testAssemblies'
-      testAssemblyVer2: 'src\x64\Release\AppInstallerCLIE2ETests\AppInstallerCLIE2ETests.dll'
-      runSettingsFile: 'src\x64\Release\AppInstallerCLIE2ETests\Test.runsettings'
+      testAssemblyVer2: '$(buildOutDir)\AppInstallerCLIE2ETests\AppInstallerCLIE2ETests.dll'
+      runSettingsFile: '$(buildOutDir)\AppInstallerCLIE2ETests\Test.runsettings'
       overrideTestrunParameters: '-PackagedContext true
-                                  -AICLIPackagePath $(system.defaultWorkingDirectory)\src\AppInstallerCLIPackage\bin\x64\Release
+                                  -AICLIPackagePath $(System.DefaultWorkingDirectory)\src\AppInstallerCLIPackage\bin\$(buildPlatform)\$(buildConfiguration)
                                   -AICLIPath AppInstallerCLI\winget.exe
                                   -LooseFileRegistration true
                                   -InvokeCommandInDesktopPackage true
                                   -StaticFileRootPath $(Agent.TempDirectory)\TestLocalIndex
                                   -MsixTestInstallerPath $(Build.ArtifactStagingDirectory)\AppInstallerTestMsixInstaller.msix
-                                  -ExeTestInstallerPath $(system.defaultWorkingDirectory)\src\x64\Release\AppInstallerTestExeInstaller\AppInstallerTestExeInstaller.exe
+                                  -ExeTestInstallerPath $(buildOutDir)\AppInstallerTestExeInstaller\AppInstallerTestExeInstaller.exe
                                   -PackageCertificatePath $(AppInstallerTest.secureFilePath)'
-    condition: succeededOrFailed()
-
-  - task: PublishBuildArtifacts@1
-    displayName: Publish E2E Tests Packaged x64 Log
-    inputs:
-      PathtoPublish: 'C:\Users\VssAdministrator\AppData\Local\Temp\E2ETestLogs'
-      ArtifactName: 'E2ETestPackagedx64Log'
-      publishLocation: 'Container'
-    condition: succeededOrFailed()
 
+  # TODO: This does not seem to be matching the logs for x86

```

Comment:
What does this mean by not matching?

Reply:
The logs for x86 are not under %temp%\E2ETestLogs, so it's not really copying anything. I couldn't figure out why as the tests should be copying winget logs to that path. I had left it as a TODO as it's still publishing the test results, so it's not critical.
But just now I'm realizing that the x86 used to have a different path from x64. I'll try with that.

Reply:
The logs for x86 were not there because the test cleanup was failing before getting to that part. I removed this TODO, and should have fixed the cleanup error (but still waiting for the build)